recipe/build.sh

```diff
@@ -9,7 +9,8 @@ export LDFLAGS=" -llapack ${LDFLAGS} "
 
 echo $OSX_ARCH
 if [ ${OSX_ARCH}. == . ]; then
-    export LDFLAGS+=" -fopenmp "
+    # We are not in OSX ergo linux (win build is blocked in meta.yaml)
+    export LDFLAGS+=" -L$PREFIX/lib64 -fopenmp "
```

Comment:
This shouldn't be needed but rather points to a faulty package. On conda-forge, everything should be in `lib/`.

Reply:
@xhochy any idea why this stopped working at this PR?

Reply:
And it obviously does not work ;) Thanks for reminding. So I guess my other package (spglib) is wrong and should be changed to put library into `lib/`